src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,23 +149,23 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
      */
     private function registerFileLoaders(ContainerBuilder $container)
     {
-        $prefix = DIRECTORY_SEPARATOR.'Resources'.DIRECTORY_SEPARATOR.'config'.DIRECTORY_SEPARATOR;
         $yamlResources = [];
         $xmlResources = [];
 
         foreach ($container->getParameter('kernel.bundles') as $bundle) {
             $reflectionClass = new \ReflectionClass($bundle);
-            $configDirectory = dirname($reflectionClass->getFileName()).$prefix;
-            $yamlResources = array_merge($yamlResources, glob($configDirectory.'api_resources.{yml,yaml}'));
-
-            foreach (Finder::create()->files()->in($configDirectory)->path('api_resources')->name('*.{yml,yaml}') as $file) {
-                $yamlResources[] = $file->getRealPath();
-            }
-
-            $xmlResources = array_merge($xmlResources, glob($configDirectory.'api_resources.xml'));
-
-            foreach (Finder::create()->files()->in($configDirectory)->path('api_resources')->name('*.xml') as $file) {
-                $xmlResources[] = $file->getRealPath();
+            $configDirectory = dirname($reflectionClass->getFileName()).'/Resources/config/';
+
+            try {
+                foreach (Finder::create()->files()->in($configDirectory)->path('api_resources')->name('*.{yml,yaml,xml}') as $file) {
```

Comment:
Behat is requiring both to be loaded to pass if I remember correctly. If it's not the case, there's a missing test case.